packages/react-scripts/package.json

```diff
@@ -40,6 +40,7 @@
     "cross-spawn": "4.0.0",
     "css-loader": "0.24.0",
     "detect-port": "1.0.0",
+    "dotenv": "^2.0.0",
```

Comment:
Please keep the version pinned (`2.0.0`). We’re a bit more on the conservative side about updates.